solar_system.rb

```diff
@@ -0,0 +1,262 @@
+# Create a Planet class which will represent a planet.
+# Add an initialize method which takes several arguments and uses them to set the class' instance variables.
+# Create a method that returns the Planet's attributes in an easy to read fashion.
+# Create reader methods to give a user access to read the instance variables.
+# Make your SolarSystem class take an array of Planets instead of hashes.
+
+# Verify user input for integers
+def verify(user_input)
+  until user_input.to_i.to_s == user_input
+    puts "Please enter a number: "
+    user_input = gets.chomp
+  end
+  return user_input.to_i
+end
+
+# Verify user input for "Y" or "N"
+def verify_y_n(user_input)
+  until user_input == "Y" || user_input == "N"
+    puts "Please enter Y or N: "
+    user_input = gets.chomp.upcase
+  end
+  return user_input
+end
+
+class Planets
+
+  attr_reader :name, :color, :shape, :motto, :planetary_bird, :year_length, :distance_from_the_sun
```

Comment:
Since each instance of this class represents a single planet, a better class name here might be `Planet` (no `s`). Similar to how it's `String` not `Strings`.